indium-repl.el

```diff
@@ -69,7 +69,7 @@
 
 (defun indium-repl-buffer-create ()
```

Comment:
Since it does not always create a new buffer, what about renaming the function `indium-repl-get-buffer-create`?

Reply:
Done.